extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -144,3 +144,179 @@ Pass the `origin key` to your front end. You might use this origin key to be abl
  
+## Step 3: Make a payment
+After the shopper submits their payment details or chooses to pay with a payment method that requires a redirection,
+the Adyen Web Components will generate a `makePaymentRequest`. Consult [Adyen documentation](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/payments) to see which parameters 
+are necessary for the current payment request.
+
+[Update CTP payment](https://docs.commercetools.com/http-api-projects-payments#update-payment) with `makePaymentRequest` custom field. 
+```json
+{
+  "version": "PAYMENT_VERSION",
+  "actions": [
+    {
+      "action": "setCustomField",
+      "name": "makePaymentRequest",
+      "value": "{\"riskData\":{\"clientData\":\"xxx\"},\"paymentMethod\":{\"type\":\"scheme\",\"holderName\":\"Lam Tran\",\"encryptedCardNumber\":\"adyenjs_...\",\"encryptedExpiryMonth\":\"adyenjs_...\",\"encryptedExpiryYear\":\"adyenjs_...\",\"encryptedSecurityCode\":\"adyenjs_...\"},\"billingAddress\":{\"street\":\"Test Strasse\",\"houseNumberOrName\":\"10\",\"postalCode\":\"80000\",\"city\":\"Munich\",\"stateOrProvince\":\"Bayern\",\"country\":\"DE\"},\"browserInfo\":{\"acceptHeader\":\"*/*\",\"colorDepth\":24,\"language\":\"en-US\",\"javaEnabled\":false,\"screenHeight\":1050,\"screenWidth\":1680,\"userAgent\":\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36\",\"timeZoneOffset\":-120},\"reference\":773,\"amount\":{\"currency\":\"EUR\",\"value\":1000}}"
+    }
+  ]
+}
+```
+The response from Adyen is be added to `makePaymentResponse` custom field. 
+The response contains information for the next steps of the payment process.
+For details, consult the [Adyen documentation](https://docs.adyen.com/checkout/components-web#step-3-make-a-payment)
+
+A CTP payment with `makePaymentResponse` for the case where user has to be redirected to a payment provider page for further authentication:
+```json
+{
+  "custom": {
+    "type": {
+      "typeId": "type",
+      "key": "ctp-adyen-integration-web-components-payment-type"
+    },
+    "fields": {
+      "makePaymentRequest": "{\"riskData\":{\"clientData\":\"xxx\"},\"paymentMethod\":{\"type\":\"scheme\",\"holderName\":\"Lam Tran\",\"encryptedCardNumber\":\"adyenjs_...\",\"encryptedExpiryMonth\":\"adyenjs_...\",\"encryptedExpiryYear\":\"adyenjs_...\",\"encryptedSecurityCode\":\"adyenjs_...\"},\"billingAddress\":{\"street\":\"Test Strasse\",\"houseNumberOrName\":\"10\",\"postalCode\":\"80000\",\"city\":\"Munich\",\"stateOrProvince\":\"Bayern\",\"country\":\"DE\"},\"browserInfo\":{\"acceptHeader\":\"*/*\",\"colorDepth\":24,\"language\":\"en-US\",\"javaEnabled\":false,\"screenHeight\":1050,\"screenWidth\":1680,\"userAgent\":\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36\",\"timeZoneOffset\":-120},\"reference\":773,\"amount\":{\"currency\":\"EUR\",\"value\":1000}}",
```

Comment:
Can we use the examples from Adyen documentation? It's better than this one.

Reply:
We could, but their example does not reflect reality, that's why it is so short.  When you're trying to make payment, you usually need to provide much more info. Their example doesn't work with credit cards nor Klarna, so I didn't want to provide a nice, but non functioning example.

Reply:
Not sure about the functionality part, the examples should have required data with good examples t, here please use data to show something it's not just something to run. Linking the same example with Adyen has advantages for linking them in a nice way. Because we are referring their documentations, so no context switch for data representations,  So Adyen documentation and their example data are useful also they  probably spend time for representations, someone will read those documents and think about it, it's not obvious for you because you are not focusing to understand.

TLDR; please spend time for the representations, for me it's the most important thing in the docs.

Do you think we can not improve the data below?

"https://9944328d3d5e.ngrok.io/return-
"paymentData\":\"xxx\"
"userAgent\":\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36\
"holderName\":\"Lam Tran\"

Reply:
I replaced it with more simple example. It's not the same as in Adyen docs as their example is a non-functional example. https://github.com/commercetools/commercetools-adyen-integration/pull/335/commits/05cfc52eaa423642beb881fe874bcd33856cf155